test/updater.test.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
```

Comment:
@shanel262 @matt-oc @powerbrian @CodeWriterWriter This is a top quality test. See how it uses Proxy and Sinon to fake all the bits around the module so that it can be manipulated into specific situations.